src/Lecture2.hs

```diff
@@ -191,7 +208,13 @@ verify that.
 [1,2,3,4,7]
 -}
 merge :: [Int] -> [Int] -> [Int]
-merge = error "TODO"
+merge [] [] = []
+merge x []  = x
```

Comment:
The first case with merge of empty two lists is redundant because it's already covered by the next line 🙂